cmd/flow-capture.go

```diff
@@ -333,32 +344,60 @@ func scanner() {
 				flowsToShow = flowsToShow - 1
 			}
 		} else if key == keyboard.KeyArrowRight {
-			if slices.Contains(display, pktDrop) && slices.Contains(display, dns) && slices.Contains(display, rtt) {
-				display = []string{raw}
-			} else if slices.Contains(display, raw) {
-				display = []string{standard}
-			} else if slices.Contains(display, standard) {
-				display = []string{pktDrop}
-			} else if slices.Contains(display, pktDrop) {
-				display = []string{dns}
-			} else if slices.Contains(display, dns) {
-				display = []string{rtt}
+			if slices.Contains(display, pktDropDisplay) && slices.Contains(display, dnsDisplay) && slices.Contains(display, rttDisplay) {
```

Comment:
if key == , will be cleaner as switch statement and the repeated logic it needs to be function with args I can see right and left logic can be use function with different args and same for pgUp/Down

Reply:
Sure I can improve that. Instead of using arrays I will use a enum with `all` case